code/game/objects/items/airbubble.dm

```diff
@@ -0,0 +1,479 @@
+		var/obj/item/airbubble/bag
+		if(syndie)
+			bag = new /obj/item/airbubble/syndie(get_turf(src))
+		else
+			bag = new /obj/item/airbubble(get_turf(src))
+		bag.ripped = ripped
+		bag.zipped = zipped
+		bag.internal_tank = internal_tank
+		internal_tank.forceMove(bag)
+		internal_tank = null
+		bag.w_class = ITEMSIZE_LARGE
+
+		bag.desc = "Special air bubble designed to protect people inside of it from decompressed enviroments."
+		if(syndie)
+			bag.desc += " This does not seem like a regular color scheme"
+		bag.desc += " It appears to be poorly hand folded."
+
+		if(ripped)
+			bag.icon_state = "[icon_closed]_man_folded_ripped"
+			bag.desc += " <span class='danger'>It has hole in it! Maybe you shouldn't use it!</span>"
+		else
+			bag.icon_state = "[icon_closed]_man_folded"
+		qdel(src)
+		return
+
+/obj/structure/closet/airbubble/req_breakout()
+
+	if(opened)
+		return 0 //Door's open... wait, why are you in it's contents then?
+	if(zipped)
+		return 1 // If it is zipped
+	if(breakout)
+		return -1 //Already breaking out.
+	return 0
+
+/obj/structure/closet/airbubble/proc/breakout_callback(var/mob/living/escapee)
+	if (QDELETED(escapee))
+		return FALSE
+
+	if ((world.time - last_shake) > 5 SECONDS)
+		playsound(loc, "sound/items/[pick("rip1","rip2")].ogg", 100, 1)
+		animate_shake()
+		last_shake = world.time
+
+	if (!req_breakout())
+		return FALSE
+
+	return TRUE
+
+// If we are stuck, and need to get out
+/obj/structure/closet/airbubble/mob_breakout(var/mob/living/escapee)
+	if (req_breakout() < 1)
+		return
+
+	escapee.next_move = world.time + 100
+	escapee.last_special = world.time + 100
+	escapee << "<span class='warning'>You lean on the back of \the [src] and start punching internal wall with your legs. (this will take about [breakout_time] minutes)</span>"
+	visible_message("<span class='danger'>\The [src] begins to shake violently! Something is terring it from the inside!</span>")
+
+	var/time = 360 * breakout_time * 2
+	breakout = TRUE
+
+	if (!do_after(escapee, time, act_target = src, extra_checks = CALLBACK(src, .proc/breakout_callback, escapee)))
+		breakout = FALSE
+		return
+
+	breakout = FALSE
+	escapee << "<span class='warning'>You successfully break out! Tearing the bubble's walls!</span>"
+	visible_message("<span class='danger'>\the [escapee] successfully broke out of \the [src]! Tearing the bubble's walls!</span>")
+	playsound(loc, "sound/items/[pick("rip1","rip2")].ogg", 100, 1)
+	break_open()
+	animate_shake()
+	desc += " <span class='danger'>It has hole in it! Maybe you shouldn't use it!</span>"
+
+// We are out finally, the bubble is ripped. So dump everything out from it. Especially air and user.
+/obj/structure/closet/airbubble/break_open()
+	ripped = TRUE
+	update_icon()
+	dump_contents()
+	var/datum/gas_mixture/t_air = get_turf_air()
+	t_air.merge(inside_air)
+
+// Change valve on internal tank
+/obj/structure/closet/airbubble/verb/set_internals(mob/user as mob)
```

Comment:
I should have probably noticed this before. This verb should not take arguments. The verb activator (usr) is not implicitly the first argument. If used, what happens is a menu is given to the usr to select who the user is, _if_ there are multiple candidates. eg. multiple mobs around.

SO: remove the argument for the verb. And replace internal references to `user` with `usr`. This is the one set of cases where `usr` is valid: in verbs.